include/fixed_containers/enum_utils.hpp

```diff
@@ -364,14 +365,59 @@ class SkeletalRichEnumStorageBase<RichEnumType,
 
 }  // namespace fixed_containers::rich_enums_detail
 
+template <typename RichEnumType>
+class RichEnumConstantProxy
+{
+private:
+    using BackingEnum = typename RichEnumType::BackingEnum;
+
+    BackingEnum backing_enum_{};
+
+public:
+    explicit(false) constexpr RichEnumConstantProxy(const BackingEnum& backing_enum)
```

Comment:
question from curiosity: I've noticed the pattern `explicit(false)` for a few ctors. What is the reason for it? Omitting it would have no effect since ctors are `implicit` by default, right?

Reply:
`explicit(false)` makes it explicit that the function is intended to allow implicit conversion and it is not an omission.